Cannot find module 'lodash/assign'

After installing the module with npm I get this error. I could fix it by going into the node_modules folder of this package and running npm install. But on our CI system this trick does not work.

Is there something missing i have to to or is this an issue? If so, a proper fix would be appreciated.

This is the error log I get.

```
error: Failed to lift app: { Error: Cannot find module 'lodash/assign'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/circleci/repo/node_modules/sails-hook-swagger-generator/build/lib/generators.js:36:32)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/circleci/repo/node_modules/sails-hook-swagger-generator/build/lib/swagger-doc.js:51:20)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32) code: 'MODULE_NOT_FOUND' }
```

--- a/package.json
+++ b/package.json
@@ -36,6 +36,7 @@
     "@types/mocha": "^7.0.1",
     "@types/node": "^13.7.4",
     "@types/swagger-jsdoc": "^3.0.2",
+    "@types/swagger-schema-official": "^2.0.20",
     "@typescript-eslint/eslint-plugin": "^2.19.2",
     "@typescript-eslint/parser": "^2.19.2",
     "chai": "^4.1.1",
@@ -50,7 +51,6 @@
     "typescript": "^3.8.2"
   },
   "dependencies": {
-    "@types/swagger-schema-official": "^2.0.20",
     "include-all": "^4.0.3",
     "lodash": "^4.17.4",
     "nodemon": "^2.0.2",

--- a/package.json
+++ b/package.json
@@ -36,12 +36,10 @@
     "@types/mocha": "^7.0.1",
     "@types/node": "^13.7.4",
     "@types/swagger-jsdoc": "^3.0.2",
-    "@types/swagger-schema-official": "^2.0.20",
     "@typescript-eslint/eslint-plugin": "^2.19.2",
     "@typescript-eslint/parser": "^2.19.2",
     "chai": "^4.1.1",
     "eslint": "^6.8.0",
-    "lodash": "^4.17.4",
     "mocha": "^3.5.0",
     "sails": "^1.2.4",
     "sails-disk": "^1.1.2",
@@ -52,7 +50,9 @@
     "typescript": "^3.8.2"
   },
   "dependencies": {
+    "@types/swagger-schema-official": "^2.0.20",
     "include-all": "^4.0.3",
+    "lodash": "^4.17.4",
     "nodemon": "^2.0.2",
     "request": "^2.88.2",
     "swagger-jsdoc": "^4.0.0"

